pkg/operation/botanist/component/istio/istiod_test.go

```diff
@@ -2529,6 +2521,40 @@ spec:
 				Expect(string(managedResourceSecret.Data["istio-ingress_templates_vpn-envoy-filter_test-ingress.yaml"])).To(BeEmpty())
 			})
 		})
+
+		Context("Proxy Protocol disabled", func() {
+			BeforeEach(func() {
+				for i := range igw {
+					igw[i].ProxyProtocolEnabled = false
+				}
+
+				istiod = NewIstio(
+					c,
+					renderer,
+					Values{
+						Istiod: IstiodValues{
+							Image:       "foo/bar",
+							Namespace:   deployNS,
+							TrustDomain: "foo.local",
+							Zones:       []string{"a", "b", "c"},
+						},
+						IngressGateway:       igw,
+						DNSServerAddress:     pointer.String("1.2.3.4"),
+						NodeLocalIPVSAddress: pointer.String("1.2.3.4"),
+					},
+				)
+			})
+
+			It("should successfully deploy all resources", func() {
+				Expect(c.Get(ctx, client.ObjectKeyFromObject(managedResourceSecret), managedResourceSecret)).To(Succeed())
+				Expect(managedResourceSecret.Type).To(Equal(corev1.SecretTypeOpaque))
+				Expect(managedResourceSecret.Data).To(HaveLen(42))
```

Comment:
Same as with the vpn: I would have expected this to be reduced by the number of proxy protocol specific entities, i.e. I would have expected it to be `39`.
WDYT?